eth/tracers/js/goja.go

```diff
@@ -948,3 +957,10 @@ func (l *steplog) setupObject() *goja.Object {
 	o.Set("contract", l.contract.setupObject())
 	return o
 }
+
+func min(a, b int64) int64 {
```

Comment:
there's already an implementation of `min` in `p2p/discover/common.go`. It would make sense to move it to `common` (or use a language that offers basic primitives?)